mingw-w64-godot/PKGBUILD

```
@@ -1,104 +1,102 @@
-# Maintainer: Sarah Ottinger <schalaalexiazeal@gmail.com>
+# Previous maintainer: Sarah Ottinger <schalaalexiazeal@gmail.com>
+# Update to 4.3-stable and maintained by: Felix Xing (Felixaszx) felixxing3009@outlook.com
+# Base on Arch Linux: https://gitlab.archlinux.org/archlinux/packaging/packages/godot/
 
 _realname=godot
 pkgbase=mingw-w64-${_realname}
 pkgname=${MINGW_PACKAGE_PREFIX}-${_realname}
-pkgver=3.5.1
-pkgrel=4
+pkgver=4.3
+pkgrel=1
 pkgdesc='An advanced, feature packed, multi-platform 2D and 3D game engine (mingw-w64)'
-arch=('any')
-mingw_arch=('mingw64' 'ucrt64' 'clang64')
-url="https://www.godotengine.org/"
+url='https://godotengine.org/'
 msys2_references=(
   "cpe: cpe:/a:godotengine:godot"
   "cpe: cpe:/a:godotengine:godot_engine"
 )
 license=('spdx:MIT')
+arch=('any')
+mingw_arch=('mingw64' 'ucrt64' 'clang64')
+makedepends=("make"
+             "${MINGW_PACKAGE_PREFIX}-gcc"
```

cc instead of gcc, think clang
```suggestion
             "${MINGW_PACKAGE_PREFIX}-cc"
```